e2etest/bgp_test.go

```diff
@@ -44,12 +47,56 @@ var _ = ginkgo.Describe("BGP", func() {
 	})
 
 	ginkgo.AfterEach(func() {
+		err := cs.CoreV1().ConfigMaps(testNameSpace).Delete(context.TODO(), "config", metav1.DeleteOptions{})
+		framework.ExpectNoError(err)
+
+		Eventually(func() bool {
+			_, err := cs.CoreV1().ConfigMaps(testNameSpace).Get(context.TODO(), "config", metav1.GetOptions{})
+			return apierrr.IsNotFound(err)
+		}, 1*time.Minute, 5*time.Second).Should(BeTrue(), "Failed to delete MetalLB ConfigMap")
```

Comment:
Let's hope it's quicker. Checking every minute will make this at least one minute slow.
Move to 1 sec

Reply:
it is checking every 5 seconds, for 1 minute

Reply:
Ooops :-) Can we move it to 1 second?